seaworthy/ps.py

```diff
@@ -20,8 +20,8 @@ class PsRow:
     process.
     """
 
-    pid = attr.ib(convert=int)
-    ppid = attr.ib(convert=int)
+    pid = attr.ib(converter=int)
+    ppid = attr.ib(converter=int)
```

Comment:
Should we add `>=17.4.0` to the `attrs` require?

Reply:
Good point, thanks.